logger.js

```diff
@@ -54,6 +54,7 @@ function pinoLogger (opts, stream) {
   const autoLoggingGetPath = opts.autoLogging && opts.autoLogging.getPath ? opts.autoLogging.getPath : null
   delete opts.autoLogging
 
+  const receivedMessage = opts.customReceivedMessage || function () { return 'request received' }
```

Comment:
I would prefer to have pino-http not log this message by default.

Reply:
Fixed